.eslintrc.cjs

```diff
@@ -0,0 +1,63 @@
+module.exports = {
```

Comment:
Looks like theres some bits missing from this config - can you update it to be the same as - https://github.com/DEFRA/cdp-node-backend-template/blob/main/.eslintrc.cjs